packages/telemed-ehr/app/src/pages/Schedule.tsx

```diff
@@ -147,6 +147,19 @@ export default function SchedulePage(): ReactElement {
         valueString: START_SCHEDULE,
       },
     ];
+
+    // if there is no timezone extension, add it. The default timezone is America/New_York
+    if (
+      !item?.extension?.find((ext) => {
```

Comment:
`!item?.extension?.some(ext => ext.url === timezoneExtensionUrl)`
would be better in this context as some method is more appropriate when you only need to check the presence of an element that meets a condition, as it returns a boolean directly